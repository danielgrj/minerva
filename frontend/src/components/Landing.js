import React from 'react';
import { Link } from 'react-router-dom';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import { faFacebookF, faGoogle } from '@fortawesome/free-brands-svg-icons'
import './landing.css'

export default function Landing() {
  return (
    <div className="container container--landing">
      <div className="slogan">
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quo, tempora optio aspernatur sapiente quae natus enim recusandae minima delectus ducimus! Aperiam, nisi animi tempora porro veniam facilis corrupti eaque harum!
      </div>
      <div className="identify">
        <Link to="/signup" className="signup">Signup</Link>
        <Link to="/login" className="login">Login</Link>
        <div className="social">
          <Link className="facebook"><FontAwesomeIcon icon={faFacebookF} /></Link>
          <Link className="google"><FontAwesomeIcon icon={faGoogle} /></Link>
          <a href="http://localhost:3000/api/auth/google/">Google</a>
          <a href="http://localhost:3000/api/auth/facebook/">Facebook</a>
        </div>
      </div>
    </div>
  )
}
