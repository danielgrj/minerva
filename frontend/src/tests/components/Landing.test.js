import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Landing from './../../components/Landing';

test('should render landing page correctly', () => {
  const wrapper = shallow(<Landing />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
