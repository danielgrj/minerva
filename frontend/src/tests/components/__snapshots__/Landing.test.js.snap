// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render landing page correctly 1`] = `
<div
  className="container container--landing"
>
  <div
    className="slogan"
  >
    <h1>
      Minerva
    </h1>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quo, tempora optio aspernatur sapiente quae natus enim recusandae minima delectus ducimus! Aperiam, nisi animi tempora porro veniam facilis corrupti eaque harum!
  </div>
  <div
    className="identify"
  >
    <Link
      className="signup"
      to="/signup"
    >
      Signup
    </Link>
    <Link
      className="login"
      to="/login"
    >
      Login
    </Link>
  </div>
</div>
`;
